src/current/v24.1/molt-fetch.md

```diff
@@ -558,7 +605,7 @@ The following `molt fetch` command uses `COPY FROM` to load a subset of tables f
 {% include_cached copy-clipboard.html %}
 ~~~ shell
 molt fetch \
---source 'mysql://root:password@localhost/molt' \
+--source 'mysql://root:password@localhost/molt?sslcert=.%2fsource_certs%2fclient.root.crt&sslkey=.%2fsource_certs%2fclient.root.key&sslmode=verify-full&sslrootcert=.%2fsource_certs%2fca.crt' \
```

Comment:
Might want to specify that specifying the ssl parameters is the default behavior we want here.